package.json

```diff
@@ -34,54 +34,56 @@
     "npm": ">=6.0.0"
   },
   "dependencies": {
-    "prop-types": "^15.7.2",
+    "jest-environment-node": "^27.5.1",
```

Comment:
can you explain what `jest-environment-node` is needed for (and does it need to be a dependency or dev dependency)?

Reply:
Looks like something accidental or automatic, I'll check.

Reply:
Yes, looks like automatic issue:

![image](https://user-images.githubusercontent.com/1962469/162021766-204e10af-3f77-475a-ba78-60001d676e4f.png)

Reply:
Hmm... okay, I moved it to `devDependencies`